### ISSUE TITLE
Comfyui please!

### ISSUE THREAD

**Issue by freeexit2002** [ToTheBeginning/PuLID#14]
Very good face consistency but need comfyui. Excellent work!

**Comment by huchenlei** [comment]
Just FYI, A1111 support is implemented here: https://github.com/Mikubill/sd-webui-controlnet/pull/2838.

**Comment by ToTheBeginning** [comment]
We are waiting for the community's implementation, or we could provide our version, but it would take several days to tidy the codes.

**Comment by wangfeng35** [comment]
> We are waiting for the community's implementation, or we could provide our version, but it would take several days to tidy the codes.

Looking forward to the comfyui verision.

**Comment by kft334** [comment]
This works for ComfyUI. It's just not in the manager yet.

https://github.com/ZHO-ZHO-ZHO/ComfyUI-PuLID-ZHO

Hopefully we will see this working without the need to use diffusers model format soon.

**Comment by kft334** [comment]
@wangfeng35 A ComfyUI version is listed on the main page of this repo.

It currently takes 80s+ on my 3060 to generate a single image with this using the diffusers model. Hopefully support will be extended to pytorch models as well.

**Comment by cubiq** [comment]
@ToTheBeginning native implementation for ComfyUI https://github.com/cubiq/PuLID_ComfyUI

**Comment by ToTheBeginning** [comment]
@cubiq Cool, we have included it in the Related Resources

**Comment by cubiq** [comment]
PuLID Attention masking for multiple IDs

![image](https://github.com/ToTheBeginning/PuLID/assets/427614/7d61717c-1e75-4454-866c-9888a2b9cdf1)

**Comment by KINGLIFER** [comment]
> @ToTheBeginning native implementation for ComfyUI https://github.com/cubiq/PuLID_ComfyUI

Does this version work with Flux?

**Comment by balazik** [comment]
@KINGLIFER 
No cubiq version doesn't support Flux. But mine does. It's early Alpha but it works.
https://github.com/balazik/ComfyUI-PuLID-Flux
